pep-0619.rst

```diff
@@ -23,7 +23,6 @@ items.
 
 Release Manager and Crew
 ========================
-
```

Comment:
Was this removal intentional?

A

Reply:
Hummm, no it wasn't

Reply:
Sorry I missed this as I was reviewing this on my phone :(